website/docs/r/timeboard.html.markdown

```diff
@@ -26,6 +26,11 @@ resource "datadog_timeboard" "redis" {
     request {
       q    = "avg:redis.info.latency_ms{$host}"
       type = "bars"
+      metadata_json = jsonencode({
```

Comment:
I can't seem to make this example work locally. Are you running TF 0.12? Is it possible there is a difference in syntax?

Reply:
Actually I notice the way you call functions seems to be different between 0.11 and 0.12
I also see jsonencode called out in the 0.12 changelog, is this going to be breaking - https://github.com/hashicorp/terraform/releases

Reply:
Good catch, I'll look into this and update the docs when I figure out how to make this work on TF < 0.12.

Reply:
Ok, I figured out the way to do it and amended the documentation. Please give it a go. Thanks!